Makefile.generated_files

```diff
@@ -286,6 +286,111 @@ $(DEEPCOPY_GEN):
 	touch $@
 
 #
+# Open-api generation
+#
+# Any package that wants open-api functions generated must include a
+# comment-tag in column 0 of one file of the form:
+#     // +openapi=<VALUE>
```

Comment:
also, the other tags use the binary name as the tag, so "openapi-gen" rather than "openapi"